permissions/NFC.permission

```diff
@@ -0,0 +1,25 @@
+# -*- mode: sh -*-
+
+# x-sailjail-translation-catalog = sailjail-permissions
+# x-sailjail-translation-key-description = permission-la-nfc
+# x-sailjail-description = NFC
+# x-sailjail-translation-key-long-description = permission-la-nfc_description
+# x-sailjail-long-description = Connect and use NFC devices
+
+# BEG systembus-org.sailfishos.nfc.daemon.resource
+dbus-system.talk      org.sailfishos.nfc.daemon
+dbus-system.call      org.sailfishos.nfc.daemon=org.sailfishos.nfc.*@/*
+dbus-system.broadcast org.sailfishos.nfc.daemon=org.sailfishos.nfc.*@/*
+# END systembus-org.sailfishos.nfc.daemon.resource
+# BEG systembus-org.sailfishos.nfc.settings.resource
+dbus-system.talk org.sailfishos.nfc.settings
+dbus-system.call      org.sailfishos.nfc.settings=org.sailfishos.nfc.*@/*
+dbus-system.broadcast org.sailfishos.nfc.settings=org.sailfishos.nfc.*@/*
+# END systembus-org.sailfishos.nfc.settings.resource
+# BEG systembus-org.neard.resource
+dbus-system.talk      org.neard
+dbus-system.call      org.neard=org.neard.*@/*
+dbus-system.broadcast org.neard=org.neard.*@/*
+# END systembus-org.neard.resource
+
+# FIXME: is org.sailfishos.system_nfc introspection denied on purpose or accidentally?
```

Comment:
I did not see this anywhere in the logs as warnings. No idea where this comes from.

Reply:
Leaving that behind as a reminder if that surfaces again.